system-test/logging.ts

```diff
@@ -355,6 +355,11 @@ describe('Logging', () => {
       log.write(logEntries[0], options, done);
     });
 
+    it('should write a single entry to a log Promise', async () => {
```

Comment:
Maybe something like "should write a single entry to a log as a promise"

Reply:
Done